client/landing/stepper/declarative-flow/internals/steps-repository/assign-trial-plan/index.tsx

```diff
@@ -38,9 +51,7 @@ const AssignTrialPlanStep: Step = function AssignTrialPlanStep( { navigation, da
 						{
 							apiVersion: '1.1',
 						},
-						{
-							wpcom_woocommerce_onboarding: profilerData,
-						}
+						requestBody
```

Comment:
Going to tag @andregardi here as the `requestBody` is where we need to pass in specific properties to mark the site as a entrepreneur trial site that was created through WPCOM.